service/controller/app/v1/resource/chartoperator/spec.go

```diff
@@ -0,0 +1,23 @@
+package chartoperator
+
+// Values represents the values to be passed to Helm commands related to
+// chart-operator chart.
+type Values struct {
+	ChartResource ChartResource `json:"resource"`
```

Comment:
This was bugging me already in `cluster-operator`. Can we go with the naming as we do in `apiextensions`? 

```suggestion
	ChartResource ValuesChartResource `json:"resource"`
```

Reply:
I would actually drop these types completely and use a `map[string]interface{}`. This was a suggestion from Mr @kopiczko and I like it a lot.